src/parseNode.js

```diff
@@ -344,57 +316,37 @@ type ParseNodeTypes = {
         type: "leftright",
         mode: Mode,
         loc?: ?SourceLocation,
-        value: LeftRightDelimType,
+        body: AnyParseNode[],
+        left: string,
+        right: string,
     |},
     "leftright-right": {|
         type: "leftright-right",
         mode: Mode,
         loc?: ?SourceLocation,
-        value: {|
-            type: "leftright-right",
-            value: string,
-        |},
+        delim: string,
     |},
     "mathchoice": {|
         type: "mathchoice",
         mode: Mode,
         loc?: ?SourceLocation,
-        value: {|
-            type: "mathchoice",
-            display: AnyParseNode[],
-            text: AnyParseNode[],
-            script: AnyParseNode[],
-            scriptscript: AnyParseNode[],
-        |},
+        display: AnyParseNode[],
+        text: AnyParseNode[],
+        script: AnyParseNode[],
+        scriptscript: AnyParseNode[],
     |},
     "middle": {|
         type: "middle",
         mode: Mode,
         loc?: ?SourceLocation,
-        value: {|
-            type: "middle",
-            value: string,
-        |},
+        delim: string,
     |},
     "mclass": {|
         type: "mclass",
         mode: Mode,
         loc?: ?SourceLocation,
-        value: {|
-            type: "mclass",
-            mclass: string,
-            value: AnyParseNode[],
-        |},
-    |},
-    "mod": {|
```

Comment:
It's nice that we can get rid whole nodes, but where did `mod` go?

Reply:
`mod` was replaced with macros a while back. I guess I forgot to remove the type.